--- a/src/quicktemplates2/qquickcombobox.cpp
+++ b/src/quicktemplates2/qquickcombobox.cpp
@@ -787,10 +787,6 @@
 
     popup->setVisible(false);
     popup->setParentItem(nullptr);
-    // Remove the item from the accessibility tree.
-    QQuickAccessibleAttached *accessible = accessibleAttached(popup);
-    if (accessible)
-        accessible->setIgnored(true);
 }
 
 QQuickComboBox::QQuickComboBox(QQuickItem *parent)
--- a/src/quicktemplates2/qquickcontrol.cpp
+++ b/src/quicktemplates2/qquickcontrol.cpp
@@ -857,11 +857,6 @@
 
     item->setVisible(false);
     item->setParentItem(nullptr);
-
-    // Remove the item from the accessibility tree.
-    QQuickAccessibleAttached *accessible = accessibleAttached(item);
-    if (accessible)
-        accessible->setIgnored(true);
 }
 
 void QQuickControlPrivate::updateBaselineOffset()
