--- a/src/shadowedrectangle.h
+++ b/src/shadowedrectangle.h
@@ -8,6 +8,7 @@
 
 #include <memory>
 #include <QQuickItem>
+#include <QColor>
 
 class PaintedRectangleItem;
 
--- a/src/colorutils.h
+++ b/src/colorutils.h
@@ -10,6 +10,7 @@
 #include <QObject>
 #include <QQuickItem>
 #include <QJSValue>
+#include <QDebug>
 
 /**
  * Utilities for processing items to obtain colors and information useful for
