--- a/Makefile.am
+++ b/Makefile.am
@@ -1,7 +1,7 @@
 ACLOCAL_AMFLAGS = -I m4
 DISTCHECK_CONFIGURE_FLAGS = --enable-introspection
 
-SUBDIRS = . po src dist tests
+SUBDIRS = . src dist tests
 if WITH_GTK_DOC
 SUBDIRS += docs
 endif
--- a/Makefile.in
+++ b/Makefile.in
@@ -162,7 +162,7 @@
 ETAGS = etags
 CTAGS = ctags
 CSCOPE = cscope
-DIST_SUBDIRS = . po src dist tests docs tools
+DIST_SUBDIRS = . src dist tests docs tools
 am__DIST_COMMON = $(srcdir)/Makefile.in ABOUT-NLS ar-lib compile \
 	config.guess config.rpath config.sub install-sh ltmain.sh \
 	missing
@@ -348,7 +348,7 @@
 top_srcdir = @top_srcdir@
 ACLOCAL_AMFLAGS = -I m4
 DISTCHECK_CONFIGURE_FLAGS = --enable-introspection
-SUBDIRS = . po src dist tests $(am__append_1) $(am__append_2)
+SUBDIRS = . src dist tests $(am__append_1) $(am__append_2)
 dist_noinst_DATA = LICENSE README.md
 MAINTAINERCLEANFILES = Makefile.in aclocal.m4 config.guess config.sub \
     configure depcomp install-sh ltmain.sh missing py-compile compile ar-lib \
