--- a/Modules/memory/chartWidget.cpp
+++ b/Modules/memory/chartWidget.cpp
@@ -53,7 +53,6 @@
 		paint.fillRect(1, 1, width() - 2, height() - 2, QBrush(QColor(128, 128, 128)));
 		paint.setPen(pen);
 		paint.drawRect(rect());
-		setAccessibleDescription(i18n("Not available."));
         mFreeMemoryLabel->setText(i18n("Not available."));
 
 		return false;
@@ -105,7 +104,6 @@
 
     mFreeMemoryLabel->setText(i18n("%1 free", formattedUnit(last_used)));
 
-    setAccessibleDescription(accessibleDescription.join(QLatin1Char('\n')));
 	return true;
 }
 
